src/components/sbb-clock/sbb-clock.tsx

```diff
@@ -89,132 +103,127 @@ export class SbbClock implements ComponentInterface {
+      hoursAnimationDuration +=
+        (remainingHours - hasRemainingMinutesOrSeconds) * this._secondsInAnHour;
     }
 
     if (this._clockHandSeconds) {
       this._clockHandSeconds.style.animation = '';
     }
 
     this._element.style.setProperty(
-      '--clock-hours-animation-start-angle',
-      `${Math.ceil(this._hours * 30 + this._minutes / 2)}deg`
+      '--sbb-clock-hours-animation-start-angle',
+      `${Math.ceil(this._hours * this._hoursAngle + this._minutes / 2)}deg`
     );
     this._element.style.setProperty(
-      '--clock-hours-animation-duration',
+      '--sbb-clock-hours-animation-duration',
       `${hoursAnimationDuration}s`
     );
     this._element.style.setProperty(
-      '--clock-seconds-animation-start-angle',
-      `${Math.ceil(this._seconds * (360 / 58.5))}deg`
+      '--sbb-clock-seconds-animation-start-angle',
+      `${Math.ceil(this._seconds * this._sbbSecondsAngle)}deg`
     );
     this._element.style.setProperty(
-      '--clock-seconds-animation-duration',
-      `${this._remainingSeconds}s`
+      '--sbb-clock-seconds-animation-duration',
+      `${remainingSeconds}s`
     );
 
     this._setMinutesHand();
 
     this._clockHandSeconds?.classList.add('sbb-clock__hand-seconds--initial-minute');
     this._clockHandHours?.classList.add('sbb-clock__hand-hours--initial-hour');
-    this._element.style.setProperty('--clock-animation-play-state', 'running');
+    this._element.style.setProperty('--sbb-clock-animation-play-state', 'running');
 
     this._isInitialized = true;
   }
 
+  /** Set the starting position for the minutes hand. */
   private _setMinutesHand(): void {
-    this._minutesAngle = this._minutes * 6;
     this._clockHandMinutes?.style.setProperty(
       'transform',
-      `rotateZ(${Math.ceil(this._minutesAngle)}deg)`
+      `rotateZ(${Math.ceil(this._minutes * this._minutesAngle)}deg)`
     );
   }
 
+  /** Move the hours hand to the next value. */
   private _moveHoursHand(): void {
     this._removeHoursAnimationStyles();
 
-    this._hoursAngle = Math.ceil(this._hours * 30 + this._minutes / 2);
+    let hoursAngle = Math.ceil(this._hours * this._hoursAngle + this._minutes / 2);
 
-    if (this._hoursAngle === 720) {
-      this._hoursAngle = 0;
-    } else if (this._hoursAngle > 360) {
-      this._hoursAngle -= 360;
+    if (hoursAngle >= this._fullAngle) {
+      hoursAngle -= this._fullAngle;
     }
 
-    this._clockHandHours?.style.setProperty(
-      'transform',
-      `rotateZ(${Math.ceil(this._hoursAngle)}deg)`
-    );
+    this._clockHandHours?.style.setProperty('transform', `rotateZ(${hoursAngle}deg)`);
   }
 
+  /** Move the minutes hand to the next value. */
   private _moveMinutesHand(): void {
     this._clockHandSeconds?.removeEventListener('animationend', moveMinutesHand);
 
     this._removeSecondsAnimationStyles();
 
+    this._addMinutesAndSetHands();
+
+    handMovement = setInterval(
+      this._addMinutesAndSetHands.bind(this),
```

Comment:
Nitpick; I generally prefer `() => this.method()` to `this.method.bind(this)` as it is easier to find in my mind.
What do you think?